src/ui/main.cpp

```diff
@@ -60,10 +70,12 @@ int main(int argc, char *argv[])
     } else if (translator.load(QLocale(langCode),
                                QString("%1").arg(qApp->applicationName().toLower()),
                                QString("_"),
-                               QString("%1/../../share/%2/translations")
+                               QString("%1/../share/%2/translations")
```

Comment:
Why this change? When installed, notepadqq is located at `/usr/lib/notepadqq/notepadqq-bin`, so the double `../` was ok.
Maybe add a comment next to that line, saying `// The Notepadqq binary gets installed at /usr/lib/notepadqq/notepadqq-bin`